srcpkgs/gitea/template

```diff
@@ -32,7 +32,7 @@ license="MIT"
 homepage="https://gitea.io"
 changelog="https://github.com/go-gitea/gitea/blob/master/CHANGELOG.md"
 distfiles="https://github.com/go-gitea/gitea/releases/download/v${version}/gitea-src-${version}.tar.gz"
```

Comment:
Could we just use https://dl.gitea.io/gitea/?

Reply:
changed...
good?